.coveragerc

```diff
@@ -1,5 +1,36 @@
 [run]
-omit=*curation_env*
-     *unit_test*
-     *data_steward/lib/*
-     *google-cloud-sdk*
+
+# Source packages/directories to include
+source = data_steward
+
+# Ensure all branching logic has been tested
+branch = True
+
+data_file = tests/results/coverage/.coverage
+
+# Exclude files whose paths match these patterns
+omit = *tests*
```

Comment:
nit: would probably make this slightly more specific, I could imagine "tests" as a valid substring of a real Python file in the curation pipeline.

Reply:
Good call.  Updated to `*curation/tests*`.